### ISSUE TITLE
Server Crashed.... GMX

### ISSUE THREAD

**Issue by Zorono** [KashCherry/Timer-Fix-plugin#12]
After upgrading to latest version `v1.1.1`, When `OnGameModeExit` callback is executed using `gmx` command the server crashes due to an unknown error...
but when i downgrade to `v1.1.0` it restarts using `GMX` normally...
```
SA-MP Server: 0.3.7-R3



Exception At Address: 0x0049B5A1 Module: (samp-server.exe)



Registers:

EAX: 0x022C1D8C	EBX: 0x01FB28F8	ECX: 0x00000000	EDX: 0x00000001

ESI: 0x01E1DC60	EDI: 0x01E1DE80	EBP: 0x0049B591	ESP: 0x0013FE98

EFLAGS: 0x00210202



Stack:

+0000: 0x004AB590   0x0013FF70   0x004AB1D1   0xFFFFFFFF

+0010: 0x0049CD96   0x00000001   0x013715A8   0x013715C8

+0020: 0x00000094   0x00000006   0x00000002   0x000023F0

+0030: 0x00000002   0x00000000   0x00000000   0x00000000

+0040: 0x00000000   0x00000000   0x00000000   0x00000000

+0050: 0x00000000   0x00000000   0x00000000   0x00000000

+0060: 0x00000000   0x00000000   0x00000000   0x00000000

+0070: 0x00000000   0x00000000   0x00000000   0x00000000

+0080: 0x00000000   0x00000000   0x00000000   0x00000000

+0090: 0x00000000   0x00000000   0x00000000   0x00000000

+00A0: 0x00000000   0x00000000   0x00000000   0x00000000

+00B0: 0x0049CC42   0x0049CC2A   0x0049CC2A   0x7FFDC000

+00C0: 0x00000000   0x00000000   0xC0000005   0x00000000

+00D0: 0x0013FEB8   0x0013F734   0x0013FFCC   0x0049D2F8

+00E0: 0x004BA540   0x00000000   0x0013FF94   0x75B841A8

+00F0: 0x7FFDC000   0x75B84180   0xE2FD2295   0x0013FFDC

+0100: 0x77932D01   0x7FFDC000   0x6420125B   0x00000000

+0110: 0x00000000   0x7FFDC000   0x00000000   0x00000000

+0120: 0x00000000   0x00000000   0x6420125B   0x0013FFA0

+0130: 0x00000000   0x0013FFE4   0x77952790   0x13AF9A67
```

```
[14/01/2020 | 04:25:16] Server crashed due to an unknown error
[14/01/2020 | 04:25:16] Native backtrace:
[14/01/2020 | 04:25:16] #0 71ea17cb in ?? () in plugins\timerfix.DLL
[14/01/2020 | 04:25:16] #1 71ea5782 in ?? () in plugins\timerfix.DLL
[14/01/2020 | 04:25:16] #2 00469b05 in ?? () in samp-server.exe
[14/01/2020 | 04:25:16] #3 0046d81b in ?? () in samp-server.exe
[14/01/2020 | 04:25:16] #4 0048f4af in ?? () in samp-server.exe
[14/01/2020 | 04:25:16] #5 62490858 in ?? () in C:\Windows\SYSTEM32\MSVCP140.dll
[14/01/2020 | 04:25:16] #6 62477bb1 in ?? () in C:\Windows\SYSTEM32\MSVCP140.dll
[14/01/2020 | 04:25:16] #7 62477bc3 in ?? () in C:\Windows\SYSTEM32\MSVCP140.dll
[14/01/2020 | 04:25:16] #8 62490746 in ?? () in C:\Windows\SYSTEM32\MSVCP140.dll
[14/01/2020 | 04:25:16] #9 62477c79 in ?? () in C:\Windows\SYSTEM32\MSVCP140.dll
[14/01/2020 | 04:25:16] #10 01acd733 in ?? () in plugins\discord-connector.DLL
[14/01/2020 | 04:25:16] #11 01acd8ab in ?? () in plugins\discord-connector.DLL
[14/01/2020 | 04:25:16] #12 01aced1d in ?? () in plugins\discord-connector.DLL
[14/01/2020 | 04:25:16] #13 01acd933 in ?? () in plugins\discord-connector.DLL
[14/01/2020 | 04:25:16] #14 0199d170 in ?? () in plugins\discord-connector.DLL
[14/01/2020 | 04:25:16] #15 019cf382 in ?? () in plugins\discord-connector.DLL
[14/01/2020 | 04:25:16] #16 0048d50f in ?? () in samp-server.exe
[14/01/2020 | 04:25:16] #17 0049b591 in ?? () in samp-server.exe
[14/01/2020 | 04:25:16] #18 0049b5a1 in ?? () in samp-server.exe
[14/01/2020 | 04:25:16] Registers:
[14/01/2020 | 04:25:16] EAX: 025a1d8c EBX: 022928f8 ECX: 00000000 EDX: 00000001
[14/01/2020 | 04:25:16] ESI: 02101b20 EDI: 02101b00 EBP: 0049b591 ESP: 0013fe98
[14/01/2020 | 04:25:16] EIP: 0049b5a1 EFLAGS: 00210202
[14/01/2020 | 04:25:16] Stack:
[14/01/2020 | 04:25:16] ESP+00000000: 004ab590 0013ff70 004ab1d1 ffffffff
[14/01/2020 | 04:25:16] ESP+00000020: 00000094 00000006 00000002 000023f0
[14/01/2020 | 04:25:16] ESP+00000040: 00000000 00000000 00000000 00000000
[14/01/2020 | 04:25:16] ESP+00000060: 00000000 00000000 00000000 00000000
[14/01/2020 | 04:25:16] ESP+00000080: 00000000 00000000 00000000 00000000
[14/01/2020 | 04:25:16] ESP+000000a0: 00000000 00000000 00000000 00000000
[14/01/2020 | 04:25:16] ESP+000000c0: 00000000 00000000 c0000005 00000000
[14/01/2020 | 04:25:16] ESP+000000e0: 004ba540 00000000 0013ff94 75b841a8
[14/01/2020 | 04:25:16] ESP+00000100: 77932d01 7ffd6000 21248d14 00000000
[14/01/2020 | 04:25:16] ESP+00000120: 00000000 00000000 21248d14 0013ffa0
[14/01/2020 | 04:25:16] ESP+00000140: 00000000 0013ffec 77932ccf ffffffff
[14/01/2020 | 04:25:16] ESP+00000160: 7ffd6000 00000000 78746341 00000020
[14/01/2020 | 04:25:16] ESP+00000180: 00000020 00000000 00000014 00000001
[14/01/2020 | 04:25:16] ESP+000001a0: 00000000 00000000 00000000 00000000
[14/01/2020 | 04:25:16] ESP+000001c0: 00000044 000002e0 00000260 00000000
[14/01/2020 | 04:25:16] ESP+000001e0: 0000031e 00000000 2d59495b 000008ac
[14/01/2020 | 04:25:16] ESP+00000200: 32ceeacd 00000be0 00000042 00000c24
[14/01/2020 | 04:25:16] ESP+00000220: 0000005e 00000fbc 00000368 00000000
[14/01/2020 | 04:25:16] ESP+00000240: 00000398 00000010 00000009 000000ec
[14/01/2020 | 04:25:16] ESP+00000260: 00000001 00000002 0000174c 000004b4
[14/01/2020 | 04:25:16] ESP+00000280: 00000001 00000004 00002a8c 00000314
[14/01/2020 | 04:25:16] ESP+000002a0: 00000002 00000006 00002e38 000000cc
[14/01/2020 | 04:25:16] ESP+000002c0: 00000001 00000009 00002ff4 00000028
[14/01/2020 | 04:25:16] ESP+000002e0: 00000001 64487353 0000002c 00000001
[14/01/2020 | 04:25:16] ESP+00000300: 00000001 00001598 0000002c 0000005e
[14/01/2020 | 04:25:16] ESP+00000320: 00000000 00000000 00000000 00000000
[14/01/2020 | 04:25:16] ESP+00000340: 00000038 00000000 003a0043 0057005c
[14/01/2020 | 04:25:16] ESP+00000360: 006e0069 00780053 005c0073 00000000
[14/01/2020 | 04:25:16] ESP+00000380: 00000260 00000001 f33271ba 000003c4
[14/01/2020 | 04:25:16] ESP+000003a0: 2d59495b 00000730 00000032 00000764
[14/01/2020 | 04:25:16] ESP+000003c0: 00000042 00000aa8 00000336 00000004
[14/01/2020 | 04:25:16] ESP+000003e0: 00000368 00000005 b1280544 000011a8
```

**Comment by aaamouii** [comment]
Thanks for your report.

Now, I think, this should be fixed.
https://github.com/KashCherry/Timer-Fix-plugin/releases/tag/v1.1.1-patch1

**Comment by Zorono** [comment]
Thanks dude, that new patch fixed it.
